lib/spells/BattleSpellMechanics.cpp

```diff
@@ -448,16 +448,16 @@ void BattleSpellMechanics::doRemoveEffects(ServerCallback * server, const std::v
 {
 	SetStackEffect sse;
 
-	for(auto unit : targets)
+	for(const auto *unit : targets)
 	{
 		std::vector<Bonus> buffer;
 		auto bl = unit->getBonuses(selector);
 
-		for(auto item : *bl)
+		for(const auto& item : *bl)
```

Comment:
that's a very strange mix of spacing... Any chance to unify? Per current code style `&` and `*` should actually be surrounded by spaces.

Reply:
passed via clang-format all.